content/operate/rc/cloud-integrations/vercel.md

```diff
@@ -0,0 +1,73 @@
+---
+LinkTitle: Vercel Marketplace
+Title: Create a Redis Cloud database with the Vercel integration.
+alwaysopen: false
+categories:
+- docs
+- operate
+- rc
+description: Shows how to create a Redis Cloud database with Vercel integration.
+hideListLinks: true
+weight: 66
+---
+
+[Redis Cloud is now integrated with Vercel](https://vercel.com/redis-cloud/~/integrations/products/redis-cloud), allowing you to create a new Redis database from your Vercel account and connect it to your Vercel project(s).
+
+## Create database
+
+1. Log in to your Vercel account (or create a new one).
+
+1. Navigate to the **Storage** tab and select **Create database**.
+    {{<image filename="/images/rc/vercel-storage-tab-create-db.png" alt="Vercel storage tab">}}
+
+1. Under **Storage partners**, select **View all partners**
+    {{<image filename="/images/rc/vercel-browse-storage.png" alt="Browse Storage partners">}}
+
+1. Find **Redis Cloud** and select **Continue**
+
+1. In the **Create Database** dialog, select your plan and **Continue**.
+
+    {{<image filename="/images/rc/vercel-create-db-select-plan.png" alt="Create database">}}
+
+    More configuration options are coming soon, such as region selection and multi-zone high availability.
+
+1. Enter your database name or use the leave the automatically generated name.
+
+1. Select **Create**.
+
+## Connect to your database
+
+After creation, you will see your database details. After provisioning is complete, the status will change from `Initializing` to `Available` (you may need to refresh your browser).
+
+{{<image filename="/images/rc/vercel-db-detail.png" alt="Vercel database details">}}
+
+You can use the connection string shown under **Quickstart** to [connect to your database]({{<relref "operate/rc/rc-quickstart/#connect-to-a-database">}}).
+
+## Link database to your project
+
+1. Navigate to the **Storage** tab.
+1. Find your new database in the list of your team's databases.
+1. Select **Connect Project**.
+    {{<image filename="/images/rc/vercel-list-connect-project.png" alt="List databases to connect project">}}
+1. Choose your project and environments and select **Connect**.
+    {{<image filename="/images/rc/vercel-connect-project.png" alt="Connect project">}}
+
+## Manage your database
+
+From the database details page, you can make edits to your database under **Settings**.
+
+More configuration options are coming soon, the plan changes, multi-zone high availability, and region selection.
```

Comment:
```suggestion
More configuration options are coming soon, including plan changes, multi-zone high availability, and region selection.
```
I'm not sure what this sentence is referring to - Is this list of configuration options what you can currently configure, or is it the list of options you'll be able to configure soon?This suggestion is for the latter.